### PR TITLE
Apply converters to values in table properties

### PR DESCRIPTION
**Pull request by ivan-lednev**

### CODE REVIEW

File: jbehave-core/src/main/java/org/jbehave/core/model/ExamplesTable.java [hunk 1]
```diff
@@ -160,7 +160,8 @@
     private static final Map<String, String> EMPTY_MAP = Collections.emptyMap();
     private static final String EMPTY_VALUE = "";
 
-    public static final Pattern INLINED_PROPERTIES_PATTERN = Pattern.compile("\\{(.*?[^\\\\])\\}\\s*(.*)", DOTALL);
+    public static final Pattern INLINED_PROPERTIES_PATTERN = Pattern.compile("\\{(.*?[^\\\\])\\}(?!\\s*[\\},])\\s*(.*)",
```
Comment (valfirst) [inline jbehave-core/src/main/java/org/jbehave/core/model/ExamplesTable.java]: this change should go as a separate commit and should be covered with unit tests

**Review by valfirst — APPROVED** [review]

**Review by ikalinin1 — APPROVED** [review]
